test/hotspot/jtreg/serviceability/dcmd/vm/SystemMapTest.java

```diff
@@ -28,15 +28,28 @@
 import jdk.test.lib.process.OutputAnalyzer;
 
 /*
- * @test
+ * @test id=normal
  * @summary Test of diagnostic command System.map
  * @library /test/lib
- * @requires (os.family == "linux" | os.family == "windows" | os.family == "mac")
+ * @requires (vm.gc != "Z") & (os.family == "linux" | os.family == "windows" | os.family == "mac")
  * @modules java.base/jdk.internal.misc
  *          java.compiler
  *          java.management
  *          jdk.internal.jvmstat/sun.jvmstat.monitor
- * @run testng/othervm -XX:+UsePerfData SystemMapTest
+ * @run testng/othervm -XX:+UsePerfData -XX:-UseZGC SystemMapTest
```

Comment:
Technically you don't need to disable ZGC as the requires clause means it is already not the selected GC

Reply:
Done.